Documentation/Storage-Configuration/Block-Storage-RBD/block-storage.md

```diff
@@ -206,6 +206,9 @@ Deploy csi-addons controller and enable `csi-addons` sidecar as mentioned in the
 
 ### Handling Node Loss
 
+This feature is disabled now, please refer to the [manual steps](../../Troubleshooting/ceph-csi-common-issues.md#node-loss) to recover from the node loss.
```

Comment:
Let's make this more obvious, perhaps as a warning. Also, some suggested edits...
```suggestion
!!! warning
    Automated node loss handling is currently disabled, please refer to the [manual steps](../../Troubleshooting/ceph-csi-common-issues.md#node-loss) to recover from the node loss.
    We are actively working on a new design for this feature.
```